tests/test_core.py

```diff
@@ -1914,13 +1851,13 @@ def test_biggerswitch(self):
 59899: 598995989959899
 Success!''')
 
-  @no_wasm_backend()
+  @no_wasm_backend("WebAssembly hasn't implemented computed gotos")
```

Comment:
I suggest the comment here and below shouldn't mention "WebAssembly" by itself - either "WebAssembly backend" (so clear it's not asm2wasm), or not mention it (context is clear from the decorator name anyhow).